.circleci/config.yml

```diff
@@ -79,6 +79,18 @@ jobs:
           root: ~/repo
           paths: .
 
+  backup:
+    docker:
+      - image: google/cloud-sdk
+    environment:
+    steps:
+      - run:
+          name: Backup Firestore
+          command: |
+            echo $GCLOUD_SERVICE_KEY | gcloud auth activate-service-account --key-file=-
+            gcloud --quiet config set project ${FIREBASE_PROJECT_ID}
+            gcloud firestore export gs://${GCLOUD_STORAGE_BUCKET} --collection-ids='addendums','whitelists','agreements'
```

Comment:
Is there a way to name the backup with something like "backup before deploying version: <commit hash>" so we'd know which point we'd need to roll back to in case of a problem?

Reply:
Yes, we can specify the prefix in `gcloud firestore export` command.
Do we have any way to keep the current version? the file `VERSION` or other approaches?

Reply:
Since we are deploying directly from master, I think we should tag backups using the commit sha we are about to deploy.

Reply:
There are some default variable provided by CircleCI
https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables

Reply:
We're not using a `VERSION` file, so it's hard to tell what the currently deployed version is, unless we could store and query that from Firebase? 

The version deployed is given by `CIRCLE_SHA1` (I think).

Open to better ways of doing this.

Reply:
Would `before_<CIRCLE_SHA1>` be a reasonable prefix?

Reply:
It looks to me, Let me try it.

Reply:
Looks good, the directory is named `before_9a7e43e2eba7d1356d0d9df48aea82f241268264/`